app/src/main/java/org/mozilla/focus/browser/InfoFragment.kt

```diff
@@ -32,14 +32,20 @@ class InfoFragment : EngineViewLifecycleFragment(), Session.Observer {
         val session = Session(arguments!!.getString(ARGUMENT_URL))
         session.register(this, owner = this)
 
-        webView!!.render(requireComponents.sessionManager.getOrCreateEngineSession(session))
+        val engineSession = requireComponents.sessionManager.getOrCreateEngineSession(session)
+        engineSession.disableTrackingProtection()
```

Comment:
That's a good question.

I re-added this because the initial integration PR removed this.

If I remember correctly this part is inherited from Focus, which has changed since then. I think we didn't want use tracking protection because it may break some of the linked pages - but I do not remember.

Reply:
Ah, found the issue where I added this in Focus:
https://github.com/mozilla-mobile/focus-android/issues/717

Will add a comment.